types/leaflet/index.d.ts

```diff
@@ -1840,6 +1841,10 @@ export namespace Icon {
 
     class Default extends Icon<DefaultIconOptions> {
         static imagePath?: string;
+
+        // Intended to be a private property but is required to be exposed for this bugfix by vue2-leaflet to work: https://vue2-leaflet.netlify.com/quickstart/#marker-icons-are-missing
+        _getIconUrl: (name: string) => string;
```

Comment:
It should be local because it's relevant only for `Vue Leaflet`

Reply:
I'll raise this with vue2-leaflet.

Reply:
It's not just relevant to just Vue Leaflet. issue is with @types/leaflet. Basically the types definition is missing this type `_getIconUrl`.
 
This was a valid change and should have merged instead of closing.